src/framework/uicomponents/view/popupview.h

```diff
@@ -0,0 +1,106 @@
+#ifndef POPUPVIEW_H

```

Comment:
Please, add a license to h and cpp and replace POPUPVIEW_H with MU_UICOMPONENTS_POPUPVIEW_H